packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VideoController/Slider.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Grommet, Box, RangeInput } from 'grommet'
+import counterpart from 'counterpart'
+import en from './locales/en.json'
+
+const customThemeRangeInput = {
+  global: {
+    spacing: '16px'
+  },
+  rangeInput: {
+    track: {
+      color: 'accent-2',
+      height: '6px',
+      extend: () => `border-radius: 10px`,
+      lower: {
+        color: '#F0B200'
+      },
+      upper: {
+        color: 'dark-4'
+      }
+    },
+    thumb: {
+      color: '#F0B200'
+    }
+  }
+}
+
+counterpart.registerTranslations('en', en)
+
+const Slider = ({ played, onMouseUp, onMouseDown, onChange }) => (
+  <Grommet theme={customThemeRangeInput}>
+    <Box>
+      <RangeInput
+        a11yTitle={counterpart('VideoController.scrubber')}
+        min={0}
+        max={1}
```

Comment:
VoiceOver announces the slider value as a number between 0 and 1. Maybe it should be the elapsed time?